### PR TITLE
Autogenstudio - Add GroupChat Support to UI

### PR DESCRIPTION
**Pull request by victordibia**
## Why are these changes needed?

Add support for GroupChat, and other quality of life improvements to autogenstudio

https://github.com/microsoft/autogen/assets/1547007/58e66ce1-3c10-4c5e-b45d-783121d151ec

## Related issue number

#1285  #1265 #1294 #1296 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: samples/apps/autogen-studio/README.md [hunk 1]
```diff
@@ -108,6 +110,16 @@ The agent workflow responds by _writing and executing code_ to create a python p
...
+## Contribution Guide
+
+We welcome contributions to AutoGen Studio. We recommend the following general steps to contribute to the project:
+
+- Review the overall AutoGen project [contribution guide](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing)
+- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project.
+- Please initiate a discussion on the roadmap issue or a new issue to discuss your proposed contribution.
+- Please review the autogenstudio dev branch here [dev branch](https://github.com/microsoft/autogen/tree/autogenstudio) and use as a base for your contribution. This way, your contribution will be aligned with the latest changes in the AutoGen Studio project.
+- Submit a pull request with your contribution!
+
```
Comment (gagb) [inline samples/apps/autogen-studio/README.md:122]:
```suggestion
- If you are modifying AutoGen Studio, it has its own devcontainer. See instructions in `.devcontainer/README.md` to use it
- Please use the tags `studio` for any issues, questions, and PRs related to Studio. 
- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project. Help is appreciated especially with Studio issues tagged with `help-wanted`.

```

File: samples/apps/autogen-studio/README.md [hunk 2]
```diff
@@ -108,6 +110,16 @@ The agent workflow responds by _writing and executing code_ to create a python p
 
 <!-- ![ARA](./docs/ara_console.png) -->
 
+## Contribution Guide
+
+We welcome contributions to AutoGen Studio. We recommend the following general steps to contribute to the project:
+
+- Review the overall AutoGen project [contribution guide](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing)
+- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project.
```
Comment (gagb) [inline samples/apps/autogen-studio/README.md:118]:
```suggestion
- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project. Help is appreciated especially with Studio issues tagged with `help-wanted`.
```

File: samples/apps/autogen-studio/README.md [hunk 3]
```diff
@@ -119,6 +131,10 @@ A: To reset your conversation history, you can delete the `database.sqlite` file
 **Q: Is it possible to view the output and messages generated by the agents during interactions?**
 A: Yes, you can view the generated messages in the debug console of the web UI, providing insights into the agent interactions. Alternatively, you can inspect the `database.sqlite` file for a comprehensive record of messages.
 
+**Q: Can I use Other Models with AutoGen Studio?**
```
Comment (gagb) [inline samples/apps/autogen-studio/README.md:134]:
```suggestion
**Q: Can I use other models with AutoGen Studio?**
```

File: website/blog/2023-12-01-AutoGenStudio/index.mdx [hunk 4]
```diff
@@ -189,6 +191,18 @@ As we continue to develop and refine AutoGen Studio, the road map below outlines
...
+## Contribution Guide
+
+We welcome contributions to AutoGen Studio. We recommend the following general steps to contribute to the project:
+
+- Review the overall AutoGen project [contribution guide](https://github.com/microsoft/autogen?tab=readme-ov-file#contributing)
+- Please review the AutoGen Studio [roadmap](https://github.com/microsoft/autogen/issues/737) to get a sense of the current priorities for the project.
+- Please initiate a discussion on the roadmap issue or a new issue to discuss your proposed contribution.
+- Please review the autogenstudio dev branch here [dev branch](https://github.com/microsoft/autogen/tree/autogenstudio) and use as a base for your contribution. This way, your contribution will be aligned with the latest changes in the AutoGen Studio project.
+- Submit a pull request with your contribution!
+
```
Comment (gagb) [inline website/blog/2023-12-01-AutoGenStudio/index.mdx:205]: Ensure its consistent with readme in samples.

File: website/blog/2023-12-01-AutoGenStudio/index.mdx [hunk 5]
```diff
@@ -203,5 +217,9 @@ A: Yes, you can view the generated messages in the debug console of the web UI,
 **Q: Where can I find documentation and support for AutoGen Studio?**
 A: We are constantly working to improve AutoGen Studio. For the latest updates, please refer to the [AutoGen Studio Readme](https://github.com/microsoft/autogen/tree/main/samples/apps/autogen-studio). For additional support, please open an issue on [GitHub](https://github.com/microsoft/autogen) or ask questions on [Discord](https://discord.gg/pAbnFJrkgZ).
 
+**Q: Can I use Other Models with AutoGen Studio?**
```
Comment (gagb) [inline website/blog/2023-12-01-AutoGenStudio/index.mdx:220]:
```suggestion
**Q: Can I use other models with AutoGen Studio?**

```

**Comment by victordibia** [comment]
Closing in favor of better organized PR.

**Pull request closed without merging**